clap_builder/src/builder/arg.rs

```diff
@@ -2114,7 +2114,7 @@ impl Arg {
     /// Typically this a more detailed (multi-line) message
     /// that describes the arg.
     ///
-    /// If [`Arg::help`] is not specified, this message will be displayed for `-h`.
+    /// Use [`Arg::help`] to display the same message for `-h` too.
```

Comment:
According to the table in https://github.com/clap-rs/clap/pull/5491#issuecomment-2111012338 the old text was correct